src/components/DataTable/cells/TableCellText.tsx

```diff
@@ -0,0 +1,50 @@
+import {TableCellWrapper} from './TableCellWrapper';
+import type {CellContent} from '../DataTable.types';
+
+export const TableCellText = ({value}: { value: string | CellContent | null | undefined }) => {
```

Comment:
If we go with a direct component usage, we can pass the props directly instead of using a configuration object.

```
<TableCellText leadingSlot={<Icon/>} label="cell text" trainlingSlot={}/>
```